evalml/tests/objective_tests/test_cost_benefit_matrix.py

```diff
@@ -32,8 +33,8 @@ def test_cbm_objective_automl(optimize_thresholds, X_y_binary):
     pipeline = automl.best_pipeline
     pipeline.fit(X, y)
     predictions = pipeline.predict(X, cbm)
-    assert not np.isnan(predictions.to_series()).values.any()
-    assert not np.isnan(pipeline.predict_proba(X).to_dataframe()).values.any()
+    assert not np.isnan(predictions).values.any()
+    assert not np.isnan(pipeline.predict_proba(X)).values.any()
```

Comment:
Lol go home codecov you're drunk 😆 

Was this change required by lint? Or just seemed like a good idea? I understand you'll get this and the other `objective_test_uses_automl` tests green in a separate PR.